taoswswrap/tdengine_connection.py

```diff
@@ -201,3 +207,27 @@ def run(
                 finally:
                     termination_thread = Thread(target=process.kill_and_close)
                     termination_thread.start()
+
+    def run_directly(
+        self,
+        statements: Optional[Union[str, Statement, list[Union[str, Statement]]]] = None,
+        query: Optional[str] = None,
+        retries: int = 0,
+        timeout: int = 0,
+    ):
```

Comment:
```suggestion
    ) -> QueryResult:
```

same for the `run_indirectly`